irods_builder.rpm.el9.Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM rockylinux:9
+
+SHELL ["/bin/bash", "-c"]
+
+RUN dnf update -y && \
+    dnf install -y epel-release && \
+    dnf config-manager --enable crb && \
+    rm -rf /tmp/*
+
+RUN dnf install -y \
+        git \
+        gnupg \
+        lsb_release \
+        wget \
+        rpm-build \
+    && \
+    rm -rf /tmp/*
+
+RUN rpm --import https://packages.irods.org/irods-signing-key.asc && \
+    wget -qO - https://packages.irods.org/renci-irods.yum.repo | tee /etc/yum.repos.d/renci-irods.yum.repo
+
+ARG irods_version=4.3.4-0.el9
+RUN dnf update -y && \
+    dnf install -y \
+        irods-devel-${irods_version} \
+        irods-runtime-${irods_version} \
+        irods-externals-clang13.0.1-0 \
+        irods-externals-cmake3.21.4-0 \
+        irods-externals-fmt-libcxx8.1.1-1 \
+        irods-externals-json3.10.4-0 \
+        irods-externals-jwt-cpp0.6.99.1-0 \
+        irods-externals-nanodbc-libcxx2.13.0-2 \
+        irods-externals-spdlog-libcxx1.9.2-2 \
+        libcurl-devel \
+        openssl-devel \
+        openssl \
```

Comment:
Do we need both? I'd think we would only need `openssl-devel`.

Reply:
just copied the deb version...   defer to smart people.

Reply:
It probably doesn't matter. We can always tighten up later.